modules/setting/setting.go

```diff
@@ -265,6 +270,10 @@ var (
 			Description: "Gitea (Git with a cup of tea) is a painless self-hosted Git service written in Go",
 			Keywords:    "go,git,self-hosted,gitea",
 		},
+		Explore: struct {
+			RequireSigninView bool `ini:"REQUIRE_SIGNIN_VIEW"`
+			DisableUsersPage  bool `ini:"DISABLE_USERS_PAGE"`
```

Comment:
I think maybe move this to service is better? We need a feature to disable all the user explore include APIs. A standalone setting for UI search seems not enough.

Reply:
I really think we have way too much stuff in `[service]` already - so if it's going in there it should go in `[service.explore]` I think.

Reply:
+1 on `[service.explore]`